_layouts/department.html

```diff
@@ -21,12 +22,18 @@
 {% endif %}
 
 
+{%
+  include components/department-budgets/ArrowButtons/index.html
+  fixed="true"
+  link_1="#section-plan"
```

Comment:
Avoid parameterising unnecessarily. The links don't change, just hard-code them into the HTML itself.

Reply:
Think of it like this: the arrow buttons are designed specifically for the page. They "know" they're on the department page, they don't need to be naive and parameterised to behave correctly. If we need them to live outside of the dept page AND they need to behave differently, then we can change it. Right now, we don't.

Reply:
Will do.